xbmc/windowing/WinSystem.cpp

```diff
@@ -15,6 +15,7 @@
 #include "settings/lib/Setting.h"
 #include "settings/Settings.h"
 #include "settings/SettingsComponent.h"
+#include "utils/log.h"
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
Done